tss-esapi/build.rs

```diff
@@ -12,6 +12,11 @@ fn main() {
     let supported_tss_version =
         VersionReq::parse("<4.0.0, >=2.3.3").expect("Failed to parse supported TSS version");
 
+    let hierarchy_is_esys_tr_req = VersionReq::parse(">=3.2.0").unwrap();
```

Comment:
Yes, exactly like this! But you should probably rename that so it makes sense.
```rust
has_simulator_unix_socket_ req = VersionReq::parse(">=3.2.0").unwrap();
if has_simulator_unix_socket_ req.matches(&tss_version) {
    println!("cargo:rustc-cfg=has_simulator_unix_socket")
}
```

Reply:
haha :D sorry